CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Conditionally render viewer with rights logic (DR-3645)
 - Added Playwright testing library (DR-3680)
 - Added Playwright tests for navigation menu, what is public domain and search bar, explore further for the home page (DR-3658)
+- Added tests for featured section on the homepage(DR-3658)
```

Comment:
Q: Do we want these change logs to be listed in the same order of the PRs? Not sure what the convention here for ya